### ISSUE TITLE
Add CKAN Adapter

### ISSUE THREAD

**Issue by jqnatividad** [openpermit/OpenPermit.NET#70]
CKAN is the leading open-source data portal platform used in CivicData, CivicDashboards and other solution providers.  Several jurisdictions have also opted to install in-premise CKAN installations.

**Comment by mmartin78** [comment]
Is this something you would like to work on?

**Comment by jqnatividad** [comment]
C#/.NET is not in our wheelhouse, so it will have be a low priority in the backlog if we're to work on it.

Perhaps, once the [datapackages work is done](https://github.com/ckan/ideas-and-roadmap/pull/160#discussion_r43489144), we can implement a package that can be consumed by openpermit.net.  The benefit of datapackages is that its designed to be platform-agnostic, and not CKAN specific.

https://github.com/ckan/ideas-and-roadmap/blob/datapackages-spec/specs/datapackages/README.md
